eyeshade/workers/reports.js

```diff
@@ -717,6 +719,31 @@ exports.workers = {
                                     threshold, usd)
       data = info.data
 
+      if (notifyP) {
+        for (let offset in data) {
+          const datum = data[offset]
+
+          if (datum.verified) continue
+
+          // at present, domains only
+          if (datum.publisher.indexOf(':') !== -1) continue
```

Comment:
i will run it tomorrow from the office where it can talk to the publishers-staging site.

i could add something like `supportedP = ...` are you indicating that it is hard (or impossible) to test for youtube or other channels?

Reply:
it was just a stylistic comment. we don't support youtube outreach at all for the time being

Reply:
understood, is it fair to say that youtube outreach isn't a near-term priority? that's fine, i just need to know.

Reply:
correct -- not a near term priority